src/models/chunks/YesNo.ts

```diff
@@ -0,0 +1,4 @@
+export enum YesNo {
```

Comment:
Do we really need an enum to hold yes/no values?

Reply:
Do we really need...? No, it can be implemented without enum. My reason for using enum is to reduce chance of making mistakes in situation where only two values 'yes' and 'no' are correct.

What is your concern for having an enum?

Reply:
I was just curious as the values are simply 'yes' or 'no'. Happy to stick with the enum though 👍